adoc/chapters/programming_interface.adoc

```diff
@@ -7294,6 +7321,13 @@ constant_ptr<DataT> get_pointer() const noexcept
       at the beginning of the buffer.  The return value is unspecified if the
       accessor is empty.
 
+This function may only be called from within a <<command>>.
```

Comment:
It would be nice to keep these additions in this PR, as they fix another bug in the spec.